src/herbpy/tsr/fuze.py

```diff
@@ -0,0 +1,55 @@
+import numpy
+from prpy.tsr.tsrlibrary import TSRFactory
+from prpy.tsr.tsr import *
+
+@TSRFactory('herb', 'fuze_bottle', 'point')
+def point_at_obj(robot, bottle, manip=None):
+    '''
+    @param robot The robot performing the point
+    @param bottle The bottle to point
+    @param manip The manipulator to perform the grasp. if None 
+        the active manipulator on the robot is used
+    '''
+
+    if manip is None:
+        manip_idx = robot.GetActiveManipulatorIndex()
+    else:
+        with manip.GetRobot():
+            manip.SetActive()
+            manip_idx = manip.GetRobot().GetActiveManipulatorIndex()
+		
+    # compute T_ow
+    T0_w_0 = bottle.GetTransform()
+    T0_w_1 = numpy.identity(4)
+
+    # compute T_we. Tuned once and used forever onward 
+    # Insures pointer points at object with right arm
+    arm_start =  numpy.array([[ 0.07277, -0.71135,  0.69905,  0.58575],
```

Comment:
I'm not sure how much of the above conversation this invalidates but arm_start isn't relating to the fingers and hand. I should have been more clear in my documentation. I'll explain (the slight hack) that's going on. If you all have a better method I'll gladly use it, otherwise I'll add this documentation in:

For pointing the arm has to pointing directly at the object*. So what I did was place an object at an arbitrary location (base_obj_trans) and manually moved the arm around in till it was making a direct point. The arm's transform resulting from this tuning is 'arm_start'. Setting this to be the TW_e basically sets it such that our arm points directly at the object. The TSR and specifically Bw allow the pointer to move in a sphere around the object where that sphere is of varying radius (hence this corresponds to begin closer or further from the object and pointing at it from a different angle).

Something that makes me sad is that when this gets generalized to say, ADA, there will be this initial calibration step where you define what it means for an arm to pointing directly at an object. The only slight comfort is that you should
only have to do this once at the very beginning.

This was the method Jen and I came up with for ensuring the pointer is actually
pointing directly at the object (because that what you want to end state to be).
 If another method comes to mind, that would be totally neat.

The act of curling the fingers to make the canonical pointing preshape for the f
ingers and hand would totally be handled in the action library. Right now its be
ing treated as like a post-processing step.

*caveat: One could say that exaggeration improves legibility. However whether this is true or not is super dependent on your cost functions. For example, for the cost functions used in the ROMAN paper exaggerating slightly in the direction of the goal decreased legibility. I also personally think exaggeration makes less sense wrt to pointing, so I'm going to discount it.